gap/Julia.gi

```diff
@@ -30,3 +30,33 @@ InstallMethod( MapOfFinSets,
     return MapOfFinSets( source, graph, range );
     
 end );
+
+##
+InstallOtherMethod( MorphismConstructor,
+        [ IsCategoryOfSkeletalFinSets, IsSkeletalFiniteSet, IsJuliaObject, IsSkeletalFiniteSet ], 1001,
```

Comment:
Yes, they are. Otherwise, the generic methods hit in, probably because of their higher weight.

Reply:
This will take us to debugging hell if it breaks (as in https://github.com/homalg-project/homalg_project/issues/277). I think it's time to teach categories about the filters of their object and morphism data. Then we can handle `MorphismConstructor` in a generic and clean way.

Can you simply redirect the Julia version of `MapOfFinSets` to the usual version and drop the installations for `MorphismConstructor` for now?